substrate/service/src/config.rs

```diff
@@ -98,26 +98,26 @@ impl<C: Default, G: Serialize + DeserializeOwned + BuildStorage> Configuration<C
 		configuration
 	}
 
-	/// Returns platform info
-	pub fn platform() -> String {
-		let env = Target::env();
-		let env_dash = if env.is_empty() { "" } else { "-" };
-		format!("{}-{}{}{}", Target::arch(), Target::os(), env_dash, env)
-	}
-
 	/// Returns full version string of this configuration.
 	pub fn full_version(&self) -> String {
-		Self::full_version_from_strs(self.impl_version, self.impl_commit)
-	}
-	
-	/// Returns full version string, using supplied version and commit.
-	pub fn full_version_from_strs(impl_version: &str, impl_commit: &str) -> String {
-		let commit_dash = if impl_commit.is_empty() { "" } else { "-" };
-		format!("{}{}{}-{}", impl_version, commit_dash, impl_commit, Self::platform())
+		full_version_from_strs(self.impl_version, self.impl_commit)
 	}
 
 	/// Implementation id and version.
 	pub fn client_id(&self) -> String {
 		format!("{}/v{}", self.impl_name, self.full_version())
 	}
 }
+
+/// Returns platform info
+pub fn platform() -> String {
+	let env = Target::env();
+	let env_dash = if env.is_empty() { "" } else { "-" };
+	format!("{}-{}{}{}", Target::arch(), Target::os(), env_dash, env)
```

Comment:
are these tabs or some kind of weird half-tab? github is rendering them strangely. also in the below function

Reply:
I think it's just my attempts to get these changes into GitHub - had a few problems with this today! I'll try to correct them..

Reply:
Looking around, these strange tab sizes seem common in many files on displayed on GitHub. I think it is perhaps just more noticeable in this one with the free functions.
Anyway, I tried pulling this file from Github back to my local machine and the tabs were all normal.